test/built-ins/ShadowRealm/prototype/importValue/throws-typeerror-import-throws.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2021 Chengzhong Wu. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-realmimportvalue
+description: >
+  ShadowRealm.prototype.importValue rejects with TypeError when the imported script throws.
+info: |
+  RealmImportValue ( specifierString, exportNameString, callerRealm, evalRealm, evalContext )
+
+    ...
+    17. Return ! PerformPromiseThen(innerCapability.[[Promise]], onFulfilled, callerRealm.[[Intrinsics]].[[%ThrowTypeError%]], promiseCapability).
+
+flags: [async, module]
```

Comment:
I don't think we need the module flag for these tests.

```suggestion
flags: [async]
```